app_flutter/lib/service/google_authentication.dart

```diff
@@ -39,7 +39,7 @@ class GoogleSignInService {
   final GoogleSignIn _googleSignIn;
 
   /// Whether or not the application has been signed in to.
-  bool get isAuthenticated => user != null;
+  Future<bool> get isAuthenticated => _googleSignIn.isSignedIn();
```

Comment:
If you don't want to change the getter signature you may be able to do:

```
bool get isAuthenticated => await _googleSignIn.isSignedIn();
```

Reply:
(You might need to change it to an async fn, so maybe you still need to change the signature after all, hmmm)

Reply:
```
The await expression can only be used in an async function.
Try marking the function body with either 'async' or 'async*'
```